docs/ml-classification-regression.md

```diff
@@ -111,10 +110,9 @@ Continuing the earlier example:
 [`LogisticRegressionTrainingSummary`](api/java/org/apache/spark/ml/classification/LogisticRegressionTrainingSummary.html)
 provides a summary for a
 [`LogisticRegressionModel`](api/java/org/apache/spark/ml/classification/LogisticRegressionModel.html).
-Currently, only binary classification is supported and the
-summary must be explicitly cast to
-[`BinaryLogisticRegressionTrainingSummary`](api/java/org/apache/spark/ml/classification/BinaryLogisticRegressionTrainingSummary.html). 
-Support for multiclass model summaries will be added in the future.
+In the case of binary classification, certain additional metrics are
```

Comment:
Now `BinaryLogisticRegressionTrainingSummary` inherits `LogisticRegressionSummary` so that inherits all metrics in `LogisticRegressionSummary`.  We'd better mark them in doc. :)

Reply:
What do you mean exactly? Do you propose to list the metrics in the user guide?

Reply:
Oh no. Just add a sentence to make it more clear, like:
"In the case of binary classification, `BinaryLogisticRegressionTrainingSummary` inherits all metrics in `LogisticRegressionSummary`, and certain additional metrics are added ..."
Just a minor suggestion -:)

Reply:
I'm ambivalent - I think it is fairly clear through the phrasing "additional metrics are available...", and in the API doc link provided.